package.json

```diff
@@ -98,24 +98,29 @@
     ]
   },
   "devDependencies": {
+    "@auto-it/all-contributors": "10.46.0",
+    "@auto-it/first-time-contributor": "10.46.0",
+    "@auto-it/npm": "10.46.0",
+    "@auto-it/released": "10.46.0",
     "@csstools/postcss-cascade-layers": "3.0.1",
     "@cypress/code-coverage": "3.10.7",
-    "@storybook/addon-a11y": "7.0.2",
-    "@storybook/addon-essentials": "7.0.2",
-    "@storybook/addon-interactions": "7.0.2",
-    "@storybook/addon-links": "7.0.2",
-    "@storybook/blocks": "7.0.2",
-    "@storybook/react": "7.0.2",
-    "@storybook/react-vite": "7.0.2",
-    "@storybook/test-runner": "0.10.0",
-    "@storybook/testing-library": "0.1.0",
+    "@storybook/addon-a11y": "7.1.0",
+    "@storybook/addon-essentials": "7.1.0",
+    "@storybook/addon-interactions": "7.1.0",
+    "@storybook/addon-links": "7.1.0",
+    "@storybook/blocks": "7.1.0",
+    "@storybook/react": "7.1.0",
+    "@storybook/react-vite": "7.1.0",
+    "@storybook/test-runner": "0.11.0",
+    "@storybook/testing-library": "0.2.0",
```

Comment:
@kcondon, I found that in this specific branch, I upgraded the Storybook version from 7.0 to 7.1, so that could be the source of the problem.

I'm going to upgrade Storybook to the latest version, 7.4.2, to see if that fixes the incompatibility with Safari. If the upgrade doesn't fix the error with Safari, then I can downgrade to 7.0. However, I prefer to try the upgrade first.